garak/cli.py

```diff
@@ -611,13 +611,17 @@ def worker_count_validation(workers):
                 from garak.resources.autodan import autodan_generate
 
                 try:
-                    prompt = _config.probe_options["prompt"]
-                    target = _config.probe_options["target"]
+                    probe_options = parse_cli_plugin_config("probe", args)
+                    if probe_options is None:
+                        raise ValueError("probe_options is None")
+                    prompt = probe_options["prompt"]
+                    target = probe_options["target"]
                 except Exception as e:
                     print(
                         "AutoDAN generation requires --probe_options with a .json containing a `prompt` and `target` "
                         "string"
                     )
+                    raise SystemExit(1)
```

Comment:
```suggestion
                    raise
```

The exception handler should simply re-raise.